### ISSUE TITLE
Add option for filling empty-valued tags

### ISSUE THREAD

**Issue by yunjae2** [yunjae2/tagit#13]
Empty-valued tags can be created when new tag is added but there are some data previously recorded.

Some features like below can be helpful:
```
$ tagit manage --fill-empty "a=1, b=2, c->hey, d->yo"
```
`=` is for limiting the range of the target data, and `->` is for filling empty tags.

**Comment by yunjae2** [comment]
This will be included in v0.0.8.

**Comment by yunjae2** [comment]
This functionality is included in tag update option.
```
$ tagit manage myexp --update "a=1, b=2, c->hey, d->yo"
```
The `--update` option do the following:
1) For the records having "a=1, b=2",
2) Set their tags: "c=hey, d=yo", no matter if c and d exist or not.
